ClientLib/src/main/java/org/droidplanner/services/android/impl/core/drone/autopilot/apm/solo/ArduSolo.java

```diff
@@ -172,6 +173,26 @@ public SoloComp getSoloComp() {
         return soloComp;
     }
 
+    @Override
+    public void onMavLinkMessageReceived(MAVLinkMessage message) {
```

Comment:
This doesn't fix the filtering for Solo. Solo gets msg_radio_status, not msg_radio. It's filtering the message because it's not included as a Mavlink exception message.